### ISSUE TITLE
*DRAFT: Create on screen party with animated GIF

### ISSUE THREAD

**Issue by kmeyersvt** [codeforbtv/cvoeo-app#27]
As a User, I want a mini party thrown in my honor when I meet a milestone. That is:

- first time login
- any change coming from Outcome tracker that a milestone was achieved
- recording of a deposit to their matched savings account

The party will consist of an animated GIF provided by Siuan.  When we get the data, this can be fleshed out further as to which triggers you will see.

**Comment by doub1ejack** [comment]
We should probably include some way to let the user see exactly what we're celebrating too.  This could probably be some variation on the "achievement unlocked" pattern.  Here's an example from google images:
<img src="https://quitnowapp.com/images/L10n/en/android_5.png" width=200/>
https://quitnowapp.com/images/L10n/en/android_5.png

**Comment by kmeyersvt** [comment]
Good point. We need to identify how we want to categorize these or if we want to display the entire goal or if we say 'goal completed' 'money saved' 'class attended' or 'appointment kept'.

**Comment by siuanm** [comment]
Is there to be one animated gif that is separate from the triggered accomplishment message  (like one follows the other) or are there multiple gifs with each message incorporated into a gif?

**Comment by siuanm** [comment]
I will start brainstorming some ideas and see what I can come up with.

**Comment by nfloersch** [comment]
https://stackoverflow.com/questions/35594783/how-do-i-display-an-animated-gif-in-react-native
